README.md

```diff
@@ -2,17 +2,134 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ## Getting Started
 
-First, run the development server:
+1. Install dependencies:
+  ```bash
+  yarn
+  ```
 
+2. Create a .env file, based on the .env.sample file:
+  ```bash
+  touch .env # then go fill it in!
+  ```
+
+3. Set up DynamoDB local:
+  Install Java if you don't already have it
+  Install dynamodb:
+  ```bash
+  sls dynamodb install
```

Comment:
I don't think this is meant to be sls?

Reply:
Hm, I think that's how you can install local DynamoDB. That's the command in the scripts of one of the other projects